### ISSUE TITLE
`displayName` is not set for the component of "@yamada-ui/lucide"

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#2880]
script: component-display-name
package: @yamada-ui/lucide

### Description
The component "@yamada-ui/lucide" uses `forwardRef`, but `displayName` is not set, making it difficult to see the display in [React Developer Tools](https://github.com/facebook/react/tree/main/packages/react-devtools).

### Additional Information
Set `displayName` and `__ui__` after each component definition.

For example, like this.

```tsx
const Component = forwardRef((props, ref) => {
  ...
})

Component.displayName = 'Component'
Component.__ui__ = 'Component'
```

Replace `'component'` with the name of the component. For example, for a `Select` component, it is `'Select'`.

`__ui__` is an identifier required for internal use by Yamada UI. Be sure to set it together with `displayName`.

**Comment by Itsuki54** [comment]
Do you add everything in src/icons?

**Comment by hirotomoyamada** [comment]
@Itsuki54 

Hi, thanks for the comment!

Only `Icon` components require `displayName` and `__ui__`.

**Comment by Itsuki54** [comment]
thank you i join this issue

**Comment by hirotomoyamada** [comment]
Fixed #3164
